proto/galileo/buf.yaml

```diff
@@ -0,0 +1,8 @@
+version: v1
+name: buf.build/penumbra-zone/galileo
```

Comment:
N.B. the BSR repo `buf.build/penumbra-zone/galileo` does not actually exist, so this config is kind of a lie. I don't see any reason for us to publish these protos online, so let's punt on that for now. I'll file a separate issue to clean up the proto docs on a rainy day.